LegionInvasionTimer.lua

```diff
@@ -186,43 +159,30 @@ frame:SetScript("OnEvent", function(f)
 		GameTooltip:Show()
 	end)
 	f:SetScript("OnLeave", GameTooltip_Hide)
-	local bg = f:CreateTexture(nil, "PARENT")
-	bg:SetAllPoints(f)
-	bg:SetColorTexture(0, 1, 0, 0.3)
-	f.bg = bg
-	local header = f:CreateFontString(nil, "OVERLAY", "TextStatusBarText")
-	header:SetAllPoints(f)
-	header:SetText(name)
-	f.header = header
-	f.startBar = startBar
-	f.db = legionTimerDB
+	f.bg = f:CreateTexture(nil, "PARENT")
+	f.bg:SetAllPoints(f)
+	f.bg:SetColorTexture(0, 1, 0, 0.3)
+	f.header = f:CreateFontString(nil, "OVERLAY", "TextStatusBarText")
+	f.header:SetAllPoints(f)
+	f.header:SetText(name)
 
 	if legionTimerDB.lock then
 		f:EnableMouse(false)
 		f.bg:Hide()
 		f.header:Hide()
 	end
 
-	candy.RegisterCallback(name, "LibCandyBar_Stop", function(_, bar, dontScan)
-		if not dontScan and bar == frame.bar1 and bar:Get("LegionInvasionTimer:complete") then
+	candy.RegisterCallback(name, "LibCandyBar_Stop", function(_, bar)
```

Comment:
Then surely it's worth more actually having the boss abilities being handled via DBM.

Or, create new bars for the bosses (and a config option to disable them, for those who also have DBM installed)